tests/lint/unittest_lint.py

```diff
@@ -525,6 +525,158 @@ def test_load_plugin_command_line() -> None:
     sys.path.remove(dummy_plugin_path)
 
 
+@pytest.mark.usefixtures("pop_pylintrc")
+def test_load_plugin_command_line_and_config_with_inithook_in_config_bad_value() -> None:
+    dummy_plugin_path = abspath(join(REGRTEST_DATA_DIR, "dummy_plugin"))
+    with fake_home() as home_path:
+        # construct a basic rc file that just modifies the path
+        pylintrc_file = join(home_path, "pylintrc")
+        with open(pylintrc_file, "w", encoding="utf8") as out:
+            out.writelines(
+                [
+                    "[MASTER]\n",
+                    f"init-hook=\"import sys; sys.path.append(r'{home_path}')\"\n",
+                    "load-plugins=copy_dummy\n",
+                ]
+            )
+
+        copytree(dummy_plugin_path, join(home_path, "copy_dummy"))
+
+        # To confirm we won't load this module _without_ the init hook running.
+        assert home_path not in sys.path
+
+        run = Run(
+            [
+                "--rcfile",
+                pylintrc_file,
+                "--load-plugins",
+                "copy_dummy",
+                join(REGRTEST_DATA_DIR, "empty.py"),
+            ],
+            reporter=testutils.GenericTestReporter(),
+            exit=False,
+        )
+        assert run._rcfile == pylintrc_file
+        assert home_path in sys.path
+        # The module should not be loaded
+        assert (
+            len(
+                [ch.name for ch in run.linter.get_checkers() if ch.name == "copy_dummy"]
+            )
+            == 0
+        )
```

Comment:
```suggestion
        assert (
            not any(
                ch.name == "copy_dummy" for ch in run.linter.get_checkers()
            )
        )
```

Could you refactor the other tests to use `any` as well?